<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>poky - Poky Build Tool and Metadata</title>
<meta name='generator' content='cgit v0.10.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://git.yoctoproject.org/cgit/cgit.cgi/poky/atom/meta/recipes-core/dbus/dbus-glib-0.100.2/no-examples.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/cgit.cgi/'><img src='/cgit/yocto-project-transp.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/cgit.cgi/'>index</a> : <a title='poky' href='/cgit/cgit.cgi/poky/'>poky</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='1.1_M1'>1.1_M1</option>
<option value='1.1_M2'>1.1_M2</option>
<option value='1.1_M3'>1.1_M3</option>
<option value='1.1_M4'>1.1_M4</option>
<option value='1.2_M1'>1.2_M1</option>
<option value='1.2_M2'>1.2_M2</option>
<option value='1.2_M3'>1.2_M3</option>
<option value='1.2_M4'>1.2_M4</option>
<option value='1.3_M1'>1.3_M1</option>
<option value='1.3_M2'>1.3_M2</option>
<option value='1.3_M3'>1.3_M3</option>
<option value='1.3_M4'>1.3_M4</option>
<option value='1.3_M5'>1.3_M5</option>
<option value='1.3_beta'>1.3_beta</option>
<option value='1.4_M1'>1.4_M1</option>
<option value='1.4_M3'>1.4_M3</option>
<option value='1.4_M4'>1.4_M4</option>
<option value='1.4_M5'>1.4_M5</option>
<option value='1.6_M5'>1.6_M5</option>
<option value='bernard'>bernard</option>
<option value='blinky'>blinky</option>
<option value='clyde'>clyde</option>
<option value='daisy'>daisy</option>
<option value='daisy-next'>daisy-next</option>
<option value='danny'>danny</option>
<option value='danny-next'>danny-next</option>
<option value='denzil'>denzil</option>
<option value='dora'>dora</option>
<option value='dora-toaster'>dora-toaster</option>
<option value='dylan'>dylan</option>
<option value='edison'>edison</option>
<option value='elroy'>elroy</option>
<option value='green'>green</option>
<option value='laverne'>laverne</option>
<option value='master' selected='selected'>master</option>
<option value='master-next'>master-next</option>
<option value='pinky'>pinky</option>
<option value='purple'>purple</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>Poky Build Tool and Metadata</td><td class='sub right'>git repository hosting</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/cgit.cgi/poky/about/'>about</a><a href='/cgit/cgit.cgi/poky/'>summary</a><a href='/cgit/cgit.cgi/poky/refs/'>refs</a><a href='/cgit/cgit.cgi/poky/log/meta/recipes-core/dbus/dbus-glib-0.100.2/no-examples.patch'>log</a><a class='active' href='/cgit/cgit.cgi/poky/tree/meta/recipes-core/dbus/dbus-glib-0.100.2/no-examples.patch'>tree</a><a href='/cgit/cgit.cgi/poky/commit/meta/recipes-core/dbus/dbus-glib-0.100.2/no-examples.patch'>commit</a><a href='/cgit/cgit.cgi/poky/diff/meta/recipes-core/dbus/dbus-glib-0.100.2/no-examples.patch'>diff</a><a href='/cgit/cgit.cgi/poky/stats/meta/recipes-core/dbus/dbus-glib-0.100.2/no-examples.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/cgit.cgi/poky/log/meta/recipes-core/dbus/dbus-glib-0.100.2/no-examples.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/cgit.cgi/poky/tree/'>root</a>/<a href='/cgit/cgit.cgi/poky/tree/meta'>meta</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-core'>recipes-core</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-core/dbus'>dbus</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-core/dbus/dbus-glib-0.100.2'>dbus-glib-0.100.2</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-core/dbus/dbus-glib-0.100.2/no-examples.patch'>no-examples.patch</a></div><div class='content'>blob: fbb49678282a93ffe1fdf2fd8fd95c4095dd315c (<a href='/cgit/cgit.cgi/poky/plain/meta/recipes-core/dbus/dbus-glib-0.100.2/no-examples.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
</pre></td>
<td class='lines'><pre><code>Disable compiling examples

Upstream-Status: Inappropriate [disable feature]

Signed-off-by: Kevin Tian &lt;kevin.tian@intel.com&gt;
Signed-off-by: Scott Garman &lt;scott.a.garman@intel.com&gt;

--- dbus-glib-0.70/dbus/Makefile.am.orig	2006-07-23 16:04:43.000000000 +0200
+++ dbus-glib-0.70/dbus/Makefile.am	2006-07-23 16:04:52.000000000 +0200
@@ -1,4 +1,4 @@
-SUBDIRS = . examples
+SUBDIRS = .
 
 INCLUDES=-I$(top_srcdir) $(DBUS_CFLAGS) $(DBUS_GLIB_CFLAGS) $(DBUS_GLIB_TOOL_CFLAGS) -DDBUS_COMPILATION=1 -DDBUS_LOCALEDIR=\"$(prefix)/@DATADIRNAME@/locale\" 
 
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.1 at 2014-07-21 11:20:15 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
